src/main/java/com/metamx/collections/bitmap/WrappedImmutableConciseBitmap.java

```diff
@@ -52,7 +53,10 @@ public ImmutableConciseSet getBitmap()
   @Override
   public boolean get(int value)
   {
-    return bitmap.get(value) > 0;
```

Comment:
any change we can implement this operation in ConciseSet / ImmutableConciseSet directly to avoid the overhead of creating additional concise sets?